### ISSUE TITLE
typings: not found

### ISSUE THREAD

**Issue by diahron** [angular/angular-cli#1126]
> Please provide us with the following information:
> ---------------------------------------------------------------
1. OS?  Linux Mint 18 Sarahcl. 

2.    diahron@Aspire-7736 ~ $ ng --version
/usr/local/lib/node_modules/angular-cli/node_modules/configstore/index.js:54
				throw err;
				^

Error: EACCES: permission denied, open '/home/diahron/.config/configstore/ember-cli.json'
You don't have access to this file.

    at Error (native)
    at Object.fs.openSync (fs.js:634:18)
    at Object.fs.readFileSync (fs.js:502:33)
    at Object.create.all.get (/usr/local/lib/node_modules/angular-cli/node_modules/configstore/index.js:35:26)
    at Object.Configstore (/usr/local/lib/node_modules/angular-cli/node_modules/configstore/index.js:28:44)
    at clientId (/usr/local/lib/node_modules/angular-cli/node_modules/ember-cli/lib/cli/index.js:22:21)
    at module.exports (/usr/local/lib/node_modules/angular-cli/node_modules/ember-cli/lib/cli/index.js:65:19)
    at module.exports (/usr/local/lib/node_modules/angular-cli/lib/cli/index.js:102:10)
    at /usr/local/lib/node_modules/angular-cli/bin/ng:31:5
    at /usr/local/lib/node_modules/angular-cli/node_modules/resolve/lib/async.js:46:14

node --version v6.2.1

3. I have been trying to install angular-cli for about a day or two.

4. ⠙ Installing packages for tooling via npmsh: 1: typings: not found
 unzip error /tmp/npm-4970-96090951/registry.npmjs.org/babel-core/-/babel-core-5.8.38.tgz
 error reading /tmp/npm-4970-96090951/registry.npmjs.org/q/-/q-1.4.1.tgz

5. I tried to install ng via npm. I think that just compounded problem.
> ---------------------------------------------------------------
> > Thanks! We'll be in touch soon.

**Comment by Splaktar** [comment]
Check the permissions on `/home/diahron/.config/configstore/ember-cli.json` and the directories above it. Change the owner back to yourself as needed. Looks like you must have used `sudo` earlier which could have made some of those dirs/files owned by `root`.

**Comment by filipesilva** [comment]
I think @Splaktar is correct, this looks like a permissions issue. There is more information about it here https://docs.npmjs.com/getting-started/fixing-npm-permissions

**Comment by stocksp** [comment]
I have a similar error after upgrading from the previous beta (which worked fine)
node --version v5.11.1

```
ng new ang-cli-test2
...
 create typings.json
Successfully initialized git.
⠙ Installing packages for tooling via npmsh: typings: command not found
File: sh
@angular/router@3.0.0-alpha.3 postinstall: `typings install`
spawn ENOENT
Error: @angular/router@3.0.0-alpha.3 postinstall: `typings install`
spawn ENOENT
    at ChildProcess.<anonymous> (/Users/paul/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/node_modules/npm/lib/utils/spawn.js:17:16)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:850:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
```
the failed installation is missing the 'typings' folder

**Comment by filipesilva** [comment]
@stocksp that issue is https://github.com/angular/angular-cli/issues/1092. Fixed on master, waiting for a release.

**Comment by filipesilva** [comment]
Closed as obsolete follow #1455.
